cobc/flag.def

```diff
@@ -93,11 +93,12 @@ CB_FLAG_NQ (1, "callfh", 9,
 	  "                        as external provided EXTFH interface module"))
 
 CB_FLAG_RQ (cb_ebcdic_table, 1, "ebcdic-table", 0, 14, /* cf cconv.h for all available tables */
-	_("  -febcdic-table=[DEFAULT|RESTRICTED-GC|IBM|GCOS]\tdefine EBCDIC translation table:\n"
+	_("  -febcdic-table=[DEFAULT|RESTRICTED-GC|IBM|GCOS|<filename>]\tdefine EBCDIC translation table:\n"
 	  "                        * default: translation to extended ASCII as per MF\n"
 	  "                        * restricted-gc: translation from restricted ASCII only\n"
 	  "                        * ibm: translation to restricted ASCII as per IBM\n"
-	  "                        * gcos: translation to extended ASCII as per GCOS7"))
```

Comment:
@nberth: Do you see a reason to not move ibm and gcos out to a file, too? If we do we could just name them the same placing in COB_CONFIG_DIR, possibly always using an extension like ".ttbl" (for translation table)? if the filename does not start with a perioid, slash (additional for `_WIN32`: with a `:` at the second place)

Reply:
I see no reason not to extract those from the source code indeed.

@ddeclerck Now I see the ibm translation I added was for restricted ASCII — did not recall that was the case.

Reply:
I'd place the tables as $COB_CONFIG_DIR/NAME.ttbl (so the original parameter matches) and instead of having ibm and gcos7 in there add a hint after <filename> "e. g. IBM, GCOS"

Reply:
Alright.

Examining these tables, it looks like the IBM table is a "generic" EBCDIC500 to 7-bit ASCII table. Would it make sense to have that table in a file called "ebcdic500_ascii7bit" and symlink "IBM" to that file ?
A similar remark can be made about the GCOS table (though it maps to an 8-bit ASCII where extended characters are just given consecutively).

Reply:
Also: the collation ids (CB_EBCDIC_IBM, etc...) seems no longer necessary ; should we get rid of this notion ?

Reply:
> Also: the collation ids (CB_EBCDIC_IBM, etc...) seems no longer necessary ; should we get rid of this notion ?

yes

Reply:
> Examining these tables, it looks like the IBM table is a "generic" EBCDIC500 to 7-bit ASCII table. Would it make sense to have that table in a file called "ebcdic500_ascii7bit" and symlink "IBM" to that file ? A similar remark can be made about the GCOS table (though it maps to an 8-bit ASCII where extended characters are just given consecutively).

Feel free to drop GCOS and IBM completely, giving them reasonable names in all lower-case.

Reply:
Also, what should be do about the `cob_get_collation_by_name` that we introduced a few weeks ago ? The new function `cob_load_collation` assumes it is given pointers to actual arrays, while the former expects pointers to pointers.

Reply:
It is a new function only available in the preview and rc-1, feel free to decide on the API, but ensure that the decision is final ;-)